static/css/mobile/header.less

```diff
@@ -181,9 +181,14 @@ header #auth-nav li {
 .site-title {
     a {
         color: #000;
+        display: block;
         font: 36px/32px @serif-stack;
+        overflow: hidden;
         text-decoration: none;
+        text-overflow: ellipsis;
         text-transform: uppercase;
+        white-space: nowrap;    
+        width: 240px;
```

Comment:
Is there a way to have that also work on other resolutions? ping @muffinresearch @tofumatt the CSS gurus ;)